options.go

```diff
@@ -105,6 +105,7 @@ func defaultOptions() *options {
 		delimiters:        ":=",
 		commentPrefixes:   Prefixes{"#", ";"},
 		multilinePrefixes: Prefixes{"\t", " "},
+		emptyLines:        true, // Allow empty lines in multiline values by default
```

Comment:
I guess this change should also lead to deprecation of the `AllowEmptyLines`, since this option will be ON by default and introduction of `NoEmptyLines` or something similar to bring back old behavior?